build-configs/common/theme/fonts.ts

```diff
@@ -25,5 +26,6 @@ export const commonFonts = {
   contentFontSize: '0.95rem',
   contentLineHeight: 1.4,
   standardParagraphMargin: '0.75rem',
-  subTitleFontSize: '1.8rem'
+  subTitleFontSize: '1.8rem',
+  figCaptionFontSize: '0.85rem'
```

Comment:
is just a suggested font-size that is a little bit smaller the the `contentFontSize`

Reply:
I'd make this more general, something like `hintFontSize` or similar